### ISSUE TITLE
gcc_lib contents differ 32 vs 64 bit builds

### ISSUE THREAD

**Issue by peabee** [puppylinux-woof-CE/woof-CE#1215]
2createpackages gcc_lib produces different contents for slackware-32 and slackware-64 builds - problem with package_template??

![screenshot](https://user-images.githubusercontent.com/5926865/41280535-120d372a-6e27-11e8-839d-9d8214744c72.png)

DISTRO_PKGS_SPECS-slackware64-current:
yes|gcc_dev|gcc,gcc-g++,gmp,elfutils,libmpc|exe>dev,dev,doc,nls
yes|gcc_lib|gcc,gcc-g++|exe,dev>null,doc>null,nls>null

**Comment by peabee** [comment]
woof-out_x86_64_x86_64_slackware64_current/packages-templates/gcc_lib/FIXUPHACK
is wrong - should access directory: ../gcc_dev_DEV/usr/lib64/gcc/x86_64-slackware-linux

```
#this is for slackware...
if [ -d ../gcc_dev_DEV/usr/lib64/gcc/i486-slackware-linux ];then
 DIRSPEC="`find ../gcc_dev_DEV/usr/lib64/gcc/i486-slackware-linux/ -maxdepth 1 -mindepth 1 -type d`"
 #specialdirdistro="`ls -1 usr/lib64/gcc/`" #slackware: i486-slackware-linux
 #specialdirversion="`ls -1 usr/lib64/gcc/${specialdirdistro}`"
 if [ "$DIRSPEC" != "" ];then
  SUBDIR="`echo -n "$DIRSPEC" | head -n 1 | rev | cut -f 1 -d '/' | rev`"
  #for the _DEV only...
  rm -rf usr/lib64/gcc/i486-slackware-linux/$SUBDIR/*
 fi
fi

```

**Comment by peabee** [comment]
but that change is not the answer / sufficient as it seems as though the retained directories (lib64 & share) template at:
 woof-out_x86_64_x86_64_slackware64_current/packages-templates/gcc_lib/usr
is being ignored.....meaning that bin, libexec and man are not deleted

**Comment by ninaholic** [comment]
@wdlkmpx I think adding #1172 to "testing" sounds ok. If there are errors, users can report them or we can build alternative iso with "master" too. :smile:  Plus it's easier than trying to merge recent changes with a different branch.

**Comment by wdlkmpx** [comment]
I fixed a few issues and merged #1172 (i also accidentally added other changes to that PR)
There are other fixes in previous commits

All these changes fix several known and unknown bugs.. including this issue.

This iso has all the recent changes:
www.01micko.com/wdlkmpx/slacko-6.9.9.9-20180621.iso
MD5: 08303bb2d02b241360ca6a4ed1be3c3f

There might be some new bugs, but it's up to someone else to catch them in a pokeball
